apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -33,7 +33,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
-	prommetrics "github.com/kedacore/keda/v2/pkg/prommetrics/webhook"
+	prommetrics "github.com/kedacore/keda/v2/pkg/metricscollector/webhook"
```

Comment:
```suggestion
	metricscollector "github.com/kedacore/keda/v2/pkg/metricscollector/webhook"
```

or something like this^